src/main/java/com/codecool/barbershop/barbershop/booking/BookingRepository.java

```diff
@@ -17,7 +17,13 @@
 
     Page<Booking> findAll(Pageable pageable);
     List<Booking> findAllByClient_ClientId(Long i, Sort sort);
-    int countBookingModelByBookingDateBetweenAndBookingStatus(Date start, Date end, BookingStatus status);
+
+
+
+    int countBookingsByBookingDateBetweenAndBookingStatus(Date start, Date end, BookingStatus status);
+
+
+
     List<Booking> findTop9ByBookingStatusOrderByBookingDateAsc(BookingStatus status);
```

Comment:
Do not include magic strings in the method names.  
If you feel tempted - add a parameter. In this case topBookingsCount for example